templates/docs/base/code.md

```diff
@@ -10,15 +10,11 @@ Vanilla gives you multiple ways to display code using the standard HTML elements
 
 When you refer to code inline with other text, use the <code>&lt;code></code> tag.
 
-<div class="embedded-example"><a href="/docs/examples/base/code-inline/" class="js-example">
+<div class="embedded-example"><a href="/docs/examples/base/code/inline/" class="js-example">
 View example of inline code
 </a></div>
 
-An inline `code` element can be nested within a `.p-strip--dark` element, and can also use the `.is-dark` utility class as necessary:
-
-<div class="embedded-example"><a href="/docs/examples/base/code-inline-dark/" class="js-example">
-View example of inline code
-</a></div>
+An inline `code` element can be nested within a `.p-strip--dark` element, and can also use the `.is-dark` utility class as necessary.
```

Comment:
Just drop this line as well, `p-strip--dark` is deprecated, we don't have to mention theme here. It just works.

Reply:
```suggestion
```